.gitignore

```diff
@@ -20,3 +20,4 @@ tests/results
 *.sublime-project
 *.sublime-workspace
 /.env
+/.vscode/
```

Comment:
Ideally configure a global gitignore for your own editor's specific files.

Reply:
Ah good idea, will do